CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change history for stripes-erm-components
 
+## 2022-05-11
+* ERM-1315 Deprecate `<CustomPropertiesConfigListFieldArray>`, `<CustomPropertyField>` , `<CustomPropertyFieldEdit>` and `<CustomPropertyFieldView>`
+
 ## 6.2.0 In progress
 * ERM-2095 Replace babel-eslint with @babel/eslint-parser
 * Deprecate `<Card>`, `<CustomPropertiesList>` in preparation for removal in a future release
```

Comment:
We usually place our new entries in the `In progress` section and don't make entries per date. I would place line 4 underneath line 8. Sorry for being nitpicking

Reply:
@MonireRasouli, sorry to push this back again, but I agree with Claudia above :)

Reply:
no problem, it fixed